gui/src/components/Taipy/tableUtils.tsx

```diff
@@ -71,6 +71,9 @@ const formatValue = (val: RowValue, col: ColumnDesc, formatConf: FormatConfig, n
             return val as string;
     }
 };
+const VALID_BOOLEAN_STRINGS = ["true", "1", "t", "y", "yes", "yeah", "sure"];
```

Comment:
This is kind of similar to what is done in utils/boolean.py (where by the way, choices are duplicated, which is bad).
I don't see a way to share that choices array... if you do, please do.

Reply:
Copy/paste is the best I could achieve in this case ...
I think it would be way overkill to pass the array as a window variable when constructing the index.html from template !